src/morphic.js

```diff
@@ -10578,8 +10578,7 @@ ScrollFrameMorph.prototype.adjustScrollBars = function () {
         vHeight = this.height() - this.scrollBarSize;
 
     this.changed();
-    if (this.contents.width() > this.width() +
-            MorphicPreferences.scrollBarSize) {
+    if (this.contents.width() > this.width()) {
```

Comment:
So, `MorphicPreferences.scrollBarSize` was definitely wrong since a ScrollFrame could have its own scrollbar size. However, I don't think we actually want to account for the scrollbar here. If the content is any wider than the view we should just show a scrollbar. Account for the width of scrollbar would make sense if we were always adding padding on the right/bottom.